tools/salmon/macros.xml

```diff
@@ -259,6 +259,9 @@
             <param name="writeUnmappedNames" type="boolean" truevalue="--writeUnmappedNames" falsevalue="" checked="False"
             label="Write the names of un-mapped reads to the file unmapped_names.txt."
             help=""/>
+            <param name="auxiliary_files" type="boolean" truevalue="true" falsevalue="false" checked="false" 
+            label="Generate auxiliary files."
```

Comment:
The multiqc needed one for example seems more useful than the rest ...

Reply:
Sure, I can do it. But in any case, the multiqc module requires all of them. But that's true that the ambig_info.tsv file provides interesting information even if not interested in multiqc analysis.

Reply:
All 3? Oo, what a mess. The MQC interface will look super complicated for salmon :(

Reply:
I think that those auxiliary files can be provided (at least the ambig_info.tsv) even if Salmon is not included in Multiqc; I consider it can provides interesting info.